server/src/remote.rs

```diff
@@ -477,21 +480,21 @@ impl FlightService for FlightServiceHandler {
         Err(Status::unimplemented("Not implemented."))
     }
 
-    /// Perform a specific action based on the type of the action in `request`.
-    /// Currently only the action `CommandStatementUpdate` is supported which
-    /// executes a SQL query containing a command that does not return a result.
-    /// These commands can be `CREATE TABLE table_name(...` which creates a
-    /// normal table, and `CREATE MODEL TABLE table_name(...` which creates a
-    /// model table. A model table is a specialized table for efficiently
-    /// storing multivariate time series with tags within a per field error
-    /// bound. Thus, model tables can only contain a single timestamp column,
-    /// field columns, and tag columns. If no error bound is defined for a field
-    /// column it defaults to an error bound of 0% (lossless compression).
-    ///
-    /// Examples of CREATE TABLE and CREATE MODEL TABLE commands:
-    /// * CREATE TABLE company(id INTEGER, name TEXT)
-    /// * CREATE MODEL TABLE wind_turbines(timestamp TIMESTAMP,
-    ///   field_lossless FIELD, field_lossy FIELD(5), tag_one TAG, tag_two TAG)
+    /// Perform a specific action based on the type of the action in `request`. Currently the
+    /// following actions are supported:
+    /// * `CommandStatementUpdate`: Execute a SQL query containing a command that does not
+    /// return a result. These commands can be `CREATE TABLE table_name(...` which creates a
+    /// normal table, and `CREATE MODEL TABLE table_name(...` which creates a model table.
+    /// * `FlushMemory`: Flush all data that is currently in memory to disk. This flushes both
+    /// uncompressed and compressed data in the storage engine to disk.
```

Comment:
I am afraid this could be misunderstood as writing uncompressed data to disk. Maybe we could write something like "This compresses the uncompressed data currently in memory and then flushes all compressed data in the storage engine to disk."

Reply:
Fixed.